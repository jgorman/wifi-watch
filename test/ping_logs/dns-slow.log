1520635313 DNS_LOOKUP 17:41:53 Waiting for dns lookup of ns.google.com ...
1520635314 DNS_LOOKUP 17:41:54 Waiting for dns lookup of ns.google.com ...
1520635315 DNS_LOOKUP 17:41:55 Waiting for dns lookup of ns.google.com ...
1520635317 DNS_LOOKUP 17:41:57 Waiting for dns lookup of ns.google.com ...
1520635318 DNS_LOOKUP 17:41:58 Waiting for dns lookup of ns.google.com ...
1520635319 DNS_LOOKUP 17:41:59 Waiting for dns lookup of ns.google.com ...
1520635320 DNS_LOOKUP 17:42:00 Waiting for dns lookup of ns.google.com ...
1520635320 PING ns.google.com (216.239.32.10): 56 data bytes
1520635320 64 bytes from 216.239.32.10: icmp_seq=0 ttl=46 time=34.544 ms
1520635321 64 bytes from 216.239.32.10: icmp_seq=1 ttl=46 time=31.144 ms
1520635322 64 bytes from 216.239.32.10: icmp_seq=2 ttl=46 time=61.392 ms
1520635323 64 bytes from 216.239.32.10: icmp_seq=3 ttl=46 time=29.985 ms
1520635324 64 bytes from 216.239.32.10: icmp_seq=4 ttl=46 time=152.010 ms
1520635325 64 bytes from 216.239.32.10: icmp_seq=5 ttl=46 time=149.945 ms
1520635327 Request timeout for icmp_seq 6
1520635328 Request timeout for icmp_seq 7
1520635329 Request timeout for icmp_seq 8
1520635331 
1520635331 --- ns.google.com ping statistics ---
1520635331 10 packets transmitted, 6 packets received, 40.0% packet loss
1520635331 round-trip min/avg/max/stddev = 29.985/76.503/152.010/53.705 ms
END_OF_PING
1520635332 DNS_LOOKUP 17:42:12 Waiting for dns lookup of ns.google.com ...
1520635333 DNS_LOOKUP 17:42:13 Waiting for dns lookup of ns.google.com ...
1520635334 DNS_LOOKUP 17:42:14 Waiting for dns lookup of ns.google.com ...
1520635335 DNS_LOOKUP 17:42:15 Waiting for dns lookup of ns.google.com ...
1520635336 DNS_LOOKUP 17:42:16 Waiting for dns lookup of ns.google.com ...
1520635337 DNS_LOOKUP 17:42:17 Waiting for dns lookup of ns.google.com ...
1520635338 DNS_LOOKUP 17:42:18 Waiting for dns lookup of ns.google.com ...
1520635339 PING ns.google.com (216.239.32.10): 56 data bytes
1520635339 64 bytes from 216.239.32.10: icmp_seq=0 ttl=46 time=36.815 ms
1520635340 64 bytes from 216.239.32.10: icmp_seq=1 ttl=46 time=26.418 ms
1520635341 64 bytes from 216.239.32.10: icmp_seq=2 ttl=46 time=92.105 ms
1520635342 64 bytes from 216.239.32.10: icmp_seq=3 ttl=46 time=151.393 ms
1520635344 Request timeout for icmp_seq 4
1520635345 Request timeout for icmp_seq 5
1520635346 Request timeout for icmp_seq 6
1520635347 Request timeout for icmp_seq 7
1520635348 Request timeout for icmp_seq 8
1520635350 
1520635350 --- ns.google.com ping statistics ---
1520635350 10 packets transmitted, 4 packets received, 60.0% packet loss
1520635350 round-trip min/avg/max/stddev = 26.418/76.683/151.393/49.838 ms
END_OF_PING
1520635351 DNS_LOOKUP 17:42:31 Waiting for dns lookup of ns.google.com ...
1520635352 DNS_LOOKUP 17:42:32 Waiting for dns lookup of ns.google.com ...
1520635353 DNS_LOOKUP 17:42:33 Waiting for dns lookup of ns.google.com ...
