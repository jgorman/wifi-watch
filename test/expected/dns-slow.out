17:42:00 Waiting for dns lookup of ns.google.com ...
PING ns.google.com (216.239.32.10): 56 data bytes

Time  Ping  Run Mode Failed Round Trip
----- ---- ---- ---- ------ ----------
17:42   10    6 good  40.0%   76.50 ms
17:42:18 Waiting for dns lookup of ns.google.com ...
17:42   10    4 good  60.0%   76.68 ms
17:42:33 Waiting for dns lookup of ns.google.com ...
